pymc3/math.py

```diff
@@ -13,63 +13,67 @@
 #   limitations under the License.
 
 import sys
-import theano.tensor as tt
+
+from functools import partial, reduce
+
+import numpy as np
+import scipy as sp
+import scipy.sparse
 
 # pylint: disable=unused-import
```

Comment:
this line should probably only be applied where necessary (probably `import scipy.sparse` and some of the `import theano.`) rather than to the whole block